packages/core/src/TolgeeConfig.ts

```diff
@@ -17,10 +17,31 @@ export class TolgeeConfig {
   apiKey?: string;
   inputPrefix?: string = '%-%tolgee:';
   inputSuffix?: string = '%-%';
+  /**
+   * Overrides all language settings
+   */
   forceLanguage?: string;
+  /**
+   * Overrides language auto detection
+   */
   defaultLanguage?: string = 'en';
+  /**
+   * Language which can be used for language detection
+   * and also limits which values can be stored in localStorage
+   */
   availableLanguages?: string[];
+  /**
+   * Language which is used when no translation is available for default one
+   */
   fallbackLanguage?: string;
+  /**
+   * Use localStorage for storing user language (default: true)
+   */
+  languageStore?: boolean = true;
```

Comment:
What about accept a Interface with get and set method and let user to provide their own way how to store it?

Reply:
Hmm, yes that could be a way, but I don't know if that the ideal one. i18next does this through their plugin system, which is a better way I think - as anyone can create simple package with the plugin and provide it to everyone through unified way. I would keep this for later ...

We can still have this switch for disabling the storage regardless the way it's implemented

Reply:
OK